discretize/MeshIO.py

```diff
@@ -44,6 +44,63 @@ def readCellLine(line):
         # Make the mesh
         tensMsh = TensorMesh([h1, h2, h3], x0)
         return tensMsh
+    
+    @classmethod
+    def readUBC_DC2DMesh(TensorMesh,fileName):
+        """
+        Read UBC GIF 2DTensor mesh and generate 2D Tensor mesh in simpeg
+
+        :param string fileName: path to the UBC GIF mesh file
+        :rtype: TensorMesh
+        :return: SimPEG TensorMesh 2D object
+
+        Created on Thu Nov 12 13:14:10 2015
```

Comment:
I would suggest not necessarily tracking this - all of this info is in the git log